### ISSUE TITLE
4.1.1 Parsing: does it need to be added in 'problematic for closed'?

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#427]
### Currently

4.1.1 Parsing is not listed under [Success Criteria Problematic for Closed Functionality](https://www.w3.org/TR/wcag2ict-22/#success-criteria-problematic-for-closed-functionality).

The [2013 version of WCAG2ICT](https://www.w3.org/TR/wcag2ict-20/#closed_functionality_sc) did have this SC under problematic for closed, with this bullet point:

> [4.1.1 Parsing](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-parses)—the [Intent of 4.1.1](https://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/ensure-compat-parses.html#ensure-compat-parses-intent-head) is to provide consistency so that different user agents or assistive technologies will yield the same result

### Discussion of possible update

The Task Force reached consensus on including "WCAG 2.0 and 2.1 Guidance" for SC 4.1.1, saying it applies as written with word substitution, and with notes explaining that it can apply under certain circumstances to software. So with regard to WCAG 2.0 and 2.1, SC 4.1.1 is not obsolete and removed.

Does that mean we need to consider adding 4.1.1 back into "problematic for closed"?

If we already had this discussion and made a decision, I'd be happy to accept it as-is, I just wanted to confirm.

**Comment by bruce-usab** [comment]
It causes problems for 4.1.1 is be applied to _any_ non-web software, and software with closed functionality is a subset of non-web software.  I am of the opinion that makes 4.1.1 belong in the set of SC problematic for closed.  See [notes from call](https://www.w3.org/2024/07/18-wcag2ict-minutes.html#t05) today.

**Comment by ChrisLoiselle** [comment]
@bruce-usab @maryjom I believe this is still open in that we've yet to make a decision on placement, if any. Please confirm. If there is a proposal I'm missing, please let me know where to reference and I can create a PR or add a proposal via an issue in to the Google Doc to send this to survey or vote (poll) if need be.

**Comment by bruce-usab** [comment]
@ChrisLoiselle et al., no proposals yet.

**Comment by maryjom** [comment]
PR #504 incorporates the content changes agreed in the [5 September resolution](https://www.w3.org/2024/09/05-wcag2ict-minutes#r02). Once the PR has been reviewed, this issue can be closed.

**Comment by maryjom** [comment]
Content changes have been merged. Closing.
